test/regress_dns.c

```diff
@@ -1012,12 +1012,12 @@ static struct event_base *be_connect_hostname_base = NULL;
 static void
 be_getaddrinfo_server_cb(struct evdns_server_request *req, void *data)
 {
-	int i;
+	unsigned i;
 	int *n_got_p=data;
 	int added_any=0;
 	++*n_got_p;
 
-	for (i=0;i<req->nquestions;++i) {
+	for (i = 0; i < (unsigned)req->nquestions; ++i) {
```

Comment:
I think that this one should be dropped (since at least it requires type-cast)

Reply:
Well, I can change the type of `i` to be `int` and that makes the warning go away, but that's uglier...